utils/device-observatory/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# This file is licensed under the CC0-1.0 license.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=device-observatory
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-3.0+
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://github.com/mwarning/device-observatory
+PKG_SOURCE_VERSION:=master
```

Comment:
no, this can't work, it makes a tarbcall called "device-blah-master.tar.xz" which will then always be "uptodate"

You need to use an actual version, even it's just a commit hash.